ogr/utils.py

```diff
@@ -42,16 +41,12 @@ def clone_repo_and_cd_inside(repo_name, repo_ssh_url, namespace):
     os.chdir(namespace)
     logger.debug("clone %s", repo_ssh_url)
 
-    for _ in range(CLONE_TIMEOUT):
-        proc = subprocess.Popen(["git", "clone", repo_ssh_url], stderr=subprocess.PIPE)
-        output = proc.stderr.read().decode()
-        logger.debug(
-            "Clone exited with {} and output: {}".format(proc.returncode, output)
-        )
-        if "does not exist yet" not in output:
-            break
-        sleep(1)
-    else:
+    proc = subprocess.run(
+        ["git", "clone", repo_ssh_url], stderr=subprocess.PIPE, timeout=CLONE_TIMEOUT
+    )
+    output = proc.stderr.read().decode()
```

Comment:
```suggestion
    output = proc.stderr.decode()
```

since `AttributeError: 'bytes' object has no attribute 'read'`

Reply:
I will fix it